### PR TITLE
Autogenerated print profiles.

### PR DESCRIPTION
**Pull request by pkuiper-ultimaker**
Mainly reordering of the settings.

Relates to PP-238

### CODE REVIEW

**Review by rijkvanmanen — APPROVED** [review]

File: resources/quality/ultimaker_s3/um_s3_bb0.4_PVA_Draft_Print.inst.cfg [hunk 1]
```diff
@@ -13,12 +13,12 @@ variant = BB 0.4
 
 [values]
 brim_replaces_support = False
+cool_fan_enabled = =not (support_enable and (extruder_nr == support_infill_extruder_nr))
 material_print_temperature = =default_material_print_temperature + 10
 prime_tower_enable = False
 retraction_count_max = 5
 skin_overlap = 20
+skirt_brim_minimal_length = =min(2000, 175/(layer_height*line_width))
```
Comment (nallath) [inline resources/quality/ultimaker_s3/um_s3_bb0.4_PVA_Draft_Print.inst.cfg:21]:
```suggestion
skirt_brim_minimal_length = =min(2000, 175 / (layer_height * line_width))
```

File: resources/quality/ultimaker_s3/um_s3_bb0.4_PVA_High_Quality.inst.cfg [hunk 2]
```diff
@@ -12,12 +12,12 @@ material = generic_pva
...
 
 [values]
-support_infill_sparse_thickness = =3*layer_height
 brim_replaces_support = False
+cool_fan_enabled = =not (support_enable and (extruder_nr == support_infill_extruder_nr))
 prime_tower_enable = False
 retraction_count_max = 5
+skirt_brim_minimal_length = =min(2000, 175/(layer_height*line_width))
 support_brim_enable = True
+support_infill_sparse_thickness = =3*layer_height
```
Comment (nallath) [inline resources/quality/ultimaker_s3/um_s3_bb0.4_PVA_High_Quality.inst.cfg:21]:
```suggestion
support_infill_sparse_thickness = =3 * layer_height
```

**Review by nallath — CHANGES_REQUESTED** [review]
Few minor changes. The "mistakes" were already there, but our boyscout rules says that you should try to leave it neater than how you found it :)

File: resources/quality/ultimaker_s3/um_s3_bb0.4_PVA_High_Quality.inst.cfg (hunk 2 above)
Comment (pkuiper-ultimaker) [reply resources/quality/ultimaker_s3/um_s3_bb0.4_PVA_High_Quality.inst.cfg:21]: We fixed the whitespaces :-)

**Pull request closed without merging**